src/System.Text.Json/tests/Serialization/DictionaryTests.cs

```diff
@@ -776,6 +776,17 @@ public static void HashtableFail()
         [Fact]
         public static void DeserializeDictionaryWithDuplicateKeys()
         {
+            // Non-generic IDictionary case.
+            IDictionary iDeserialize = JsonSerializer.Parse<IDictionary>(@"{""Hello"":""World"", ""Hello"":""NewValue""}");
```

Comment:
nit: Why is the IDictionary local called `iDeserialize`? Should it instead be `iDictionary`?

Reply:
because i have named all other variables as deserialize in this as well as in other pr